src/index.ts

```diff
@@ -165,6 +165,11 @@ export interface ProjectOptions {
    */
   builds?: ProjectBuildOptions[];
 
+  /**
+   * Tag all build with `basePath: true`.
```

Comment:
How about:

  Set `basePath: true` on all builds. See that option for more details.

Reply:
Updated!